tests/bench.py

```diff
@@ -42,11 +43,14 @@ def big_graph_path(tmpdir_factory, big_graph):
     return tmp_path
 
 
-def test_write(benchmark, tmp_path, big_graph):
+@pytest.mark.parametrize(
+    "write_func", [geff_nx.write_nx]
+)  # , geff_rx.write_rx])  # we cannot test rustworkx because the big_graph is a NX graph
```

Comment:
Theoretically, we could write the next graph read it in with rx and then write it out again with rx... All that prep work can be excluded from the benchmarking time

Reply:
This is okay to merge without benchmarking (Morgan and I discussed and agreed)